src/applications/Job.java

```diff
@@ -63,16 +63,43 @@ boolean moveToNextMachine(MachineShopSimulator machineShopSimulator, SimulationR
 	        return false;
 	    } else {// theJob has a next task
 	            // get machine for next task
-	        int p = ((Task) getTaskQ().getFrontElement()).getMachine();
+	        int index = ((Task) getTaskQ().getFrontElement()).getMachine();
```

Comment:
I like this change, this method was a bit hard to read. One thing you could do is extract the long call into its own method to further improve readability.